styles/custom.scss

```diff
@@ -101,6 +113,16 @@ $theme-colors: (
   }
 }
 
+@media (min-width: 1200px) {
+  .container,
+  .container-lg,
+  .container-md,
+  .container-sm,
+  .container-xl {
+    max-width: 1140px !important;
```

Comment:
the upgrades changed the default width of the containers so that the content was stretching across the entire page instead of having a good amount of white space on the left and right like the website currently has, so i added this to revert back to how the website looked before.